examples/account_management/reject_merchant_center_link.rb

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Demonstrates how to reject a Merchant Center link request.
+#
+# Prerequisite: You need to have access to a Merchant Center account. You can
+# find instructions to create a Merchant Center account here:
+# https://support.google.com/merchants/answer/188924.
+#
+# To run this example, you must use the Merchant Center UI or the Content API
+# for Shopping to send a link request between your Merchant Center and
+# Google Ads accounts.
+
+require 'optparse'
+require 'google/ads/google_ads'
+
+def reject_merchant_center_links(customer_id, merchant_center_account_id)
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  merchant_center_link_service = client.service.merchant_center_link
+
+  # Rejects a pending link request or unlinks an enabled link for a Google Ads
+  # account with customer_id from a Merchant Center account with
+  # merchant_center_account_id.
+  response = merchant_center_link_service.list_merchant_center_links(
+    customer_id: customer_id,
+  )
+
+  # Checks if there is a link for the Merchant Center account we are
+  # looking for.
+  # If the Merchant Center link is pending, reject it by removing the link.
+  # If the Merchant Center link is enabled, unlink Merchant Center from
+  # Google Ads by removing the link.
+  # In both cases, the remove action is the same.
+  # There is only one MerchantCenterLink object for a given Google Ads
+  # account and Merchant Center account, so we can just detect the first one.
+  link_to_remove = response.merchant_center_links.detect {|link| link.id == merchant_center_account_id.to_i}
+
+  if !link_to_remove.nil?
+    puts "Found the link to remove:"
+    puts "\t Link resource name '#{link_to_remove.resource_name}', " \
+      "link status: #{link_to_remove.status}"
+
+    remove_merchant_center_link(
+    client,
```

Comment:
Nit: Indent the arguments two more spaces.

Reply:
Done.